src/plugins/home/public/application/components/usecase_overview/search_use_case_app.tsx

```diff
@@ -14,28 +14,32 @@ import {
   ContentManagementPluginStart,
   SEARCH_OVERVIEW_PAGE_ID,
 } from '../../../../../content_management/public';
+import { SEARCH_USE_CASE_ID } from '../../../../../../core/public';
 
 interface Props {
   contentManagement: ContentManagementPluginStart;
 }
 
 export const SearchUseCaseOverviewApp = ({ contentManagement }: Props) => {
   const {
-    services: { workspaces, chrome },
+    services: { chrome },
   } = useOpenSearchDashboards<CoreStart>();
 
-  const currentWorkspace = useObservable(workspaces.currentWorkspace$);
-
-  const title = i18n.translate('home.usecase.search.title', { defaultMessage: 'Search overview' });
+  const currentNavGroup = useObservable(chrome.navGroup.getCurrentNavGroup$());
+  const isSearchUseCase = currentNavGroup?.id === SEARCH_USE_CASE_ID;
 
   useEffect(() => {
+    const title = i18n.translate('home.searchOverview.title', { defaultMessage: 'Overview' });
+    const titleWithUseCase = i18n.translate('home.searchOverview.titleWithUseCase', {
+      defaultMessage: 'Search Overview',
+    });
     const breadcrumbs: EuiBreadcrumb[] = [
       {
-        text: currentWorkspace?.name || title,
+        text: isSearchUseCase ? title : titleWithUseCase,
```

Comment:
Maybe add a comment here for the business logic.

Reply:
sure, comments added.